### ISSUE TITLE
Use a stable sort algorithm

### ISSUE THREAD

**Issue by nuragic** [bebraw/reactabular#59]
Hi!

```Array.prototype.sort()``` is NOT stable... https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort :cry: 

That is causing unexpected results when sorting... Luckily we can use ```_.sortBy()``` instead! https://lodash.com/docs#sortBy :smile: 

It's ok for you to make the switch?

Thanks!

**Comment by bebraw** [comment]
Sure! PR is welcome.

**Comment by nuragic** [comment]
Ok! Working on it :)

**Comment by nuragic** [comment]
@bebraw I've just launched `npm test` (without making any change) and I get:

![captura de pantalla 2015-07-13 a las 14 38 49](https://cloud.githubusercontent.com/assets/1586378/8649697/f700bf12-296c-11e5-8a28-30c818791e09.png)

:confused:

**Comment by bebraw** [comment]
@nuragic That's something Jest related likely... I nuked my deps and reinstalled them and the tests worked for me. I suppose that's one more reason to drop Jest from this project.

Can you try to replace the sorting function with this?

```
var sortBy = require('lodash').sortBy;

var sort = (data, column) => {
    if(!column) {
        return data;
    }

    var property = column.property;

    var ret = sortBy(data.concat(), (o) => o[property]);

    if(column.sort < 0) {
        ret.reverse();
    }

    return ret;
};
```

If this works well for you, we can go with it.

**Comment by eviltoylet** [comment]
I've seen that error before and I think it has to do with the version of node and jest. I don't think it's compatible with 13 yet.

**Comment by nuragic** [comment]
@bebraw I have a simpler one, basically:

```js
_.sortByOrder(data, [property], [column.sort]);
```

I'll push it today if I'm able to solve that weird jest error...

P.S. multi-column ordering would be very nice to have later :)

**Comment by bebraw** [comment]
@nuragic Ah, cool! That should be more effective too. :)

I agree on multi-column ordering. I wonder if that's possible with the current setup.

**Comment by bebraw** [comment]
Implemented in `0.6.5` (currently in beta).
